shared/templates/static/ansible/set_password_hashing_algorithm_libuserconf.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel,multi_platform_fedora
```

Comment:
hmm, this change doesn't seem to fit the commit message. perhaps the file that you reomved rhel from was not staged to be committed?